### PR TITLE
✅ Add tests with constraint with duplicates

### PR DESCRIPTION
**Pull request by blegat**
Inspired from @rdeits test in https://github.com/JuliaOpt/MathOptInterfaceMosek.jl/issues/41

### CODE REVIEW

File: src/Test/UnitTests/constraints.jl [hunk 1]
```diff
@@ -123,6 +123,72 @@ function solve_affine_interval(model::MOI.ModelLike, config::TestConfig)
 end
 unittests["solve_affine_interval"] = solve_affine_interval
 
+# Taken from https://github.com/JuliaOpt/MathOptInterfaceMosek.jl/issues/41
+"""
+    solve_duplicate_terms_scalar_affine(model::MOI.ModelLike,
+                                        config::TestConfig)
+
+Add an `ScalarAffineFunction`-in-`LessThan` constraint with duplicate terms in
```
Comment (mlubin) [inline src/Test/UnitTests/constraints.jl]: s/an/a/
